dynatrace/src/main/java/com/dynatrace/opentelemetry/metric/DynatraceMetricExporter.java

```diff
@@ -226,6 +217,16 @@ protected CompletableResultCode export(
     return CompletableResultCode.ofSuccess();
   }
 
+  private void logExportingError(InputStream errorStream, int code) throws IOException {
+    if (errorStream != null) {
```

Comment:
Nit: could invert the if to reduce nesting.

Reply:
IMO, this doesn't really change a whole lot since we would have to use a `return;` in order to actually reduce nesting, which makes the code more complicated to read.